.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -46,4 +53,5 @@
   - Described upgrades (or lack thereof) into `UPGRADING.md`
 - [ ] *Repository maintenance*
   - Any updates to `README.md`
-  - Captured changes into `CHANGELOG.md`
\ No newline at end of file
+  - Captured changes into `CHANGELOG.md`
+- [ ] Confluence documentation written or updated where appropriate
```

Comment:
Can you be more specific - what documentation exists at a PR level?

(remember that we will also have checklists for Stories and Epics, which will cover broader considerations)

Reply:
very good points, i have some questions:
- usually for me, the PR is referring to a particular ticket. is a ticket and a story the same thing for us?
- as an engineer i have not yet dealt with epics, so if i had to write confluence docs to document some engineering implementation, i would probably want to be reminded to do it at this point.

"what documentation exists at a PR level?"

i am not imagining it existing at a PR level, but i am imagining parts of the system being explained in confluence documentation. for example - client fee groups might have documentation which explains how the implementation works:
- client fee groups are implemented as a special case of portfolio groups
- the way client fee groups differ from usual portfolio groups is ... ... ...
- the collections and properties in the db to do with client fee groups are ... ... ...
- the usual way fees are calculated is ... ... ..
- client fee groups affect fees by ... ...

if we have documentation that looks similar to that:
- an individual ticket might need to create a new doc page
- an individual ticket might effect a change of how it works, so it might need the confluence doc to be updated

hence why i have included that checkbox. that being said i know we might now want to have docs that work that way